### PR TITLE
Greg/node/setup

### PR DESCRIPTION
**Pull request by GregTheGreek**

### CODE REVIEW

File: src/types/service.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type UBID = number[];
+export type Addr = number[];
+
+// TOOD fix this
+export const DISCOVERY = [0];
```
Comment (decanus) [inline src/types/service.ts]: add new line, I'd add this to the `discovery` file too.

**Review by decanus — CHANGES_REQUESTED** [review]

**Comment by decanus** [comment]
@GregTheGreek I'd like to see some tests

**Comment by GregTheGreek** [comment]
> @GregTheGreek I'd like to see some tests

fine

File: src/types/service.ts (hunk 1 above)
Comment (GregTheGreek) [reply src/types/service.ts]: > add new line

What?

>I'd add this to the discovery file too.

It's imported?

**Review by ec2 — DISMISSED** [review]
lgtm. Not sure what dean wants w.r.t. newlines?

File: src/types/service.ts (hunk 1 above)
Comment (decanus) [reply src/types/service.ts]: File doesn't have a new line at the end of it, all others do.

I wouldn't import, i'd put it there so its all in one

**Review by decanus — APPROVED** [review]

**Pull request merged**